include/cutlass/fast_math.h

```diff
@@ -246,7 +246,7 @@ template <typename value_t>
 CUTLASS_HOST_DEVICE value_t find_log2(value_t x) {
   int a = int(31 - clz(x));
```

Comment:
I'm a bit surprised that you didn't have to change this line, as `clz(x)` returns `value_t` (see the `clz` function template immediately above this).